platform/viewer/cypress/integration/common/OHIFCornerstoneToolbar.spec.js

```diff
@@ -0,0 +1,509 @@
+      //Click on button
+      cy.get('.toolbar-button:nth-child(10)').as('cineBtn')
+        .click();
+        //Vefiry if cine control overlay is being displayed
+        cy.get('.cine-controls').as('cineControls')
+          .should('be.visible');
+
+        //Test PLAY button
+        cy.get('[title="Play / Stop"]')
+          .click()
+          .wait(100)
+          .click();
+        
+        const overlaySeriesInformation = 'div.ViewportOverlay > div.bottom-left.overlay-element > div';
+        let expectedText = 'Img: 1 1/26';  
+        cy.get(overlaySeriesInformation)
+          .should('not.have.text', expectedText);
+
+        //Test SKIP TO FIRST IMAGE button
+        cy.get('[title="Skip to first Image"]')
+          .click()
+        cy.get(overlaySeriesInformation)
+          .should('contain.text', expectedText);
+
+        //Test NEXT IMAGE button
+        cy.get('[title="Play Image"]') //Title is wrong and was reported on bug #995: https://github.com/OHIF/Viewers/issues/995
+          .click()
+          expectedText = 'Img: 2 2/26'; 
+        cy.get(overlaySeriesInformation)
+          .should('contain.text', expectedText);  
+
+        //Test SKIP TO LAST IMAGE button
+        cy.get('[title="Skip, to last Image"]') //Title is wrong and was reported on bug #995: https://github.com/OHIF/Viewers/issues/995
+          .click()
+        expectedText = 'Img: 27 26/26';  
+        cy.get(overlaySeriesInformation)
+          .should('contain.text', expectedText);
+
+        //Test PREVIOUS IMAGE button
+        cy.get('[title="Previous Image"]')
+          .click()
+        expectedText = 'Img: 26 25/26'; 
+        cy.get(overlaySeriesInformation)
+          .should('contain.text', expectedText);  
+
+        //Click on Cine button
+        cy.get('@cineBtn')
+        .click();
+        //Vefiry if cine control overlay is hidden
+        cy.get('@cineControls')
+          .should('not.be.visible'); 
+    });
+    
+
+    it('checks if More button will prompt a modal with secondary tools', () => {
+      cy.get('.expandableToolMenu')
+        .click();
+      //Verify if overlay is displayed
+      cy.get('.tooltip-toolbar-overlay')
+        .should('be.visible');
+
+      let iconName;
+        //Click on one of the secondary tools from the overlay
+      cy.get('.tooltip-inner > :nth-child(1)')
+        .click()
+        .then(($magnifyBtn) =>{
+          cy.wrap($magnifyBtn)
+            .should('have.class', 'active')
+            .find('svg').then(($icon)=>{
+                iconName = $icon.text();
+            })
+        })
+
+        //Check if More button is active and if it has same icon as the secondary tool selected
+        cy.get('.expandableToolMenu')
+        .click()
+        .then(($moreBtn) =>{
+          cy.wrap($moreBtn)
+            .should('have.class', 'active')
+            .find('svg').should(($icon2)=>{
+              expect($icon2.text()).to.contain(iconName)
```

Comment:
You should be able to use something like `.contains()` to prevent drilling down to the text:
- https://docs.cypress.io/api/commands/contains.html#Syntax

Basically: `$moreBtn.contains(iconName)`